app/controllers/new_gestionnaire/procedures_controller.rb

```diff
@@ -85,5 +178,57 @@ def redirect_to_avis_if_needed
         redirect_to avis_index_path
       end
     end
+
+    def procedure_presentation
+      @procedure_presentation ||= current_gestionnaire.procedure_presentation_for_procedure_id(params[:procedure_id])
```

Comment:
Je pense que la memoization n'est pas nécessaire car active record cache par défault la query

Reply:
Oui mais en termes de perf c'est toujours mieux d'après http://gavinmiller.io/2013/basics-of-ruby-memoization/